### PR TITLE
Bug 558587 implement parameterized customer export

### PR DESCRIPTION
**Pull request by eparovyshnaya**
- append documentation

Signed-off-by: elena.parovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

**Comment by eparovyshnaya** [comment]
This PR is going to close [558587](https://bugs.eclipse.org/bugs/show_bug.cgi?id=558587)

File: tests/org.eclipse.passage.loc.report.core.tests/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -10,4 +10,4 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Require-Bundle: org.junit;bundle-version="4.12.0",
  org.eclipse.passage.loc.report.core;bundle-version="0.1.0",
  org.eclipse.osgi;bundle-version="3.15.100",
- org.eclipse.passage.lic.users
+ org.eclipse.passage.lic.users;bundle-version="0.5.100"
```
Comment (ruspl-afed) [inline tests/org.eclipse.passage.loc.report.core.tests/META-INF/MANIFEST.MF]: Does this bundle exist in the previous version?

File: bundles/org.eclipse.passage.loc.report.core/src/org/eclipse/passage/loc/report/internal/core/ExportService.java [hunk 2]
```diff
@@ -18,16 +18,32 @@
...
 	void exportCustomersForProducts(Set<String> products, Path target) throws ReportException;
 
+	/**
+	 * It is required to install a {@linkplain CustomerStorage} to the service prior
+	 * any export action is asked as it is the base source of information to be
+	 * exported.
+	 * 
+	 * @since 0.1
+	 */
 	void installCustomerStorage(CustomerStorage stogage);
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.report.core/src/org/eclipse/passage/loc/report/internal/core/ExportService.java]: Oh no, please. Let's return some prepared service if needed

Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.report.core/src/org/eclipse/passage/loc/report/internal/core/ExportService.java]: Do you mean `install` must not be void?

File: tests/org.eclipse.passage.loc.report.core.tests/META-INF/MANIFEST.MF (hunk 1 above)
Comment (eparovyshnaya) [reply tests/org.eclipse.passage.loc.report.core.tests/META-INF/MANIFEST.MF]: Not shure I've got the idea of question. Currently `org.eclipse.passage.lic.users` has `0.5.100` version.

**Pull request merged**